tests/Integration/Behaviour/Features/Context/CartRuleFeatureContext.php

```diff
@@ -154,6 +154,27 @@ public function cartRuleNamedIsRestrictedToCarrierNamed($cartRuleName, $carrierN
         Cache::clear();
     }
 
+    /**
+     * @Given /^cart rule "(.+)" is restricted to cheapest product$/
```

Comment:
"cheapest products" is a bit vague 🤔 can we improve it ?

Reply:
From behat context I see it actually is clear, so maybe it only needs an explanation ?

Reply:
this is a product specification: on cart rule configuration page, you can set the option "Exclude discounted products"

Reply:
cheapest products is different from discounted products 🤔 😉

Reply:
or 
"Apply a discount to
-Order (without shipping)
-Specific product
-Cheapest product"

Reply:
@matks sorry I pressed "comment" a bit too soon

Reply:
OK it makes sense then 👌